.github/CONTRIBUTING.md

```diff
@@ -55,16 +56,16 @@ If you've found a bug and wish to fix it, the first thing to do is
 
 If you've created a brand new App and wish to contribute it, the steps to do so are as follows.
 
+1. Create a new [issue](https://github.com/Splunk-SOAR-Apps/phawslambda/issues/new?assignees=&labels=&template=new_repo_request.md&title=) in our ```.github``` repo to request a new repository to be created for your app.
 1. [Fork](https://guides.github.com/activities/forking/) the project
-1. Create a branch (following our [Conventions](https://github.com/phantomcyber/phantom-apps/blob/next/.github/CONVENTIONS.md)))
-1. Create a new directory/folder for your App (again following the [Conventions](https://github.com/phantomcyber/phantom-apps/blob/next/.github/CONVENTIONS.md)).
-1. Add your app code to the folder. Ensure no other folders are affected.
+1. Create a branch (following our [Conventions](https://github.com/splunk-soar-apps/.github/blob/next/.github/CONVENTIONS.md)))
+1. Push your app code to the branch you created. 
 1. **Thoroughly** test your code for the new App. See the [Automated Checks](#automated-checks) section for information about basic automated checks we provide for all apps.
     <!-- 1. Ensure your new app has a [TESTING](https://about:blank) document for the community and our developers. -->
-1. Add your name to the contributors list in the app JSON! [Example](https://github.com/phantomcyber/phantom-apps/pull/488/commits/a02e345ce48e56bcb8711d1c5c4e40dd6e62fd11?diff=split&w=1)
-1. Perform a [pull request](https://help.github.com/articles/using-pull-requests/) to the [Next](https://github.com/phantomcyber/phantom-apps/tree/next) branch. Please ensure your pull request adheres to the guidelines mentioned in [PULL REQUEST TEMPLATE](https://github.com/phantomcyber/phantom-apps/blob/next/.github/pull_request_template.md).
+1. Add your name to the contributors list in the app JSON! [Example](https://github.com/splunk-soar-apps/.github/pull/488/commits/a02e345ce48e56bcb8711d1c5c4e40dd6e62fd11?diff=split&w=1)
```

Comment:
this example PR commit link doesn't exist

Reply:
this should actually still reference the old phantom-apps repo, to provide an example of a contributors list